code/game/machinery/computer/chef_orders/chef_order.dm

```diff
@@ -71,7 +70,7 @@
 	var/datum/orderable_item/wanted_item = locate(params["target"]) in order_datums
 	switch(action)
 		if("cart_set")
-			grocery_list[wanted_item] = params["amt"]
+			grocery_list[wanted_item] = clamp(params["amt"], 0, 50)
```

Comment:
Why 50?

Reply:
Should be 20, but the javascript file seems to hint that it can be as high as 20.